h2o-py/h2o/h2o.py

```diff
@@ -436,6 +439,7 @@ def import_file(path=None, destination_frame=None, parse=True, header=0, sep=Non
         directory.
     :param skipped_columns: an integer list of column indices to skip and not parsed into the final frame from the import file.
     :param custom_non_data_line_markers: If a line in imported file starts with any character in given string it will NOT be imported. Empty string means all lines are imported, None means that default behaviour for given format will be used
+    :param quotechar: A hint for the parser which character to expect as quoting character. None (default) means autodetection.
```

Comment:
can we add to those docstrings that it accepts only None (default=Auto), single quote, double quote?

Reply:
added